src/main/groovy/org/pih/warehouse/api/StockMovement.groovy

```diff
@@ -205,6 +211,15 @@ class StockMovement implements Validateable{
         ]
     }
 
+    Comment getRecentComment() {
+        if (shipmentComments?.size() > 0) {
+            return shipmentComments?.sort({ a, b ->
+                b.dateCreated <=> a.dateCreated
```

Comment:
shouldn't we look at `lastUpdated` instead of `dateCreated`? Maybe I don't know the scope very well, but I see that the `compareTo` method in `Comment` looks at the `lastUpdated` date.

If we do want to look at `lastUpdated`, a titbit: you could just write `.sort()`, since we have `compareTo` overwritten.

Reply:
yeah, this is up for debate what do we consider "the most recent comment" in my opinion it's based on date created since that's usually how it is sorted in many comment sections and messengers.